docs/migration/updating.md

```diff
@@ -0,0 +1,121 @@
+# Updating
+
+- [Update Strategies](#update-strategies)
+- [Backups](#backups)
+
+## Update Strategies
+
+There are three options on how to leverage the update assistant on agent startup:
+
+1. Manually instantiating the update assistant on agent startup
```

Comment:
No they're mostly doing the same steps. 2 is maybe the most performant because you skip the initialization of the update assistant?

We can add some tests in the future to benchmark this process

Reply:
Cool--I figured they were pretty similar, but I was curious if there was something that I had missed here. Thanks 👍